Naming conflict when importing `ga_operation`

I'm having trouble generating the client for appengine v1 and I found the issue is that there is a naming conflict with `ga_operation`

In `client.py`, `ga_operation` has a naming conflict.
```
from google.api_core import operation as ga_operation
```
conflicts with
```
from google.cloud.appengine_admin_v1.types import operation as ga_operation
```

```
__________________________________ ERROR collecting tests/unit/gapic/appengine_admin_v1/test_applications.py __________________________________
tests/unit/gapic/appengine_admin_v1/test_applications.py:38: in <module>
    from google.cloud.appengine_admin_v1.services.applications import (
google/cloud/appengine_admin_v1/__init__.py:18: in <module>
    from .services.applications import ApplicationsClient
google/cloud/appengine_admin_v1/services/applications/__init__.py:18: in <module>
    from .client import ApplicationsClient
google/cloud/appengine_admin_v1/services/applications/client.py:78: in <module>
    class ApplicationsClient(metaclass=ApplicationsClientMeta):
google/cloud/appengine_admin_v1/services/applications/client.py:392: in ApplicationsClient
    ) -> ga_operation.Operation:
E   AttributeError: module 'google.cloud.appengine_admin_v1.types.operation' has no attribute 'Operation'
```

--- a/gapic/schema/metadata.py
+++ b/gapic/schema/metadata.py
@@ -115,11 +115,12 @@ class Address:
             return '_'.join(
                 (
                     ''.join(
-                        ''.join(
-                            [partial_name[0] for partial_name in i.split("_")]
-                        )
-                        for i in self.package
-                        if i != self.api_naming.version
+                        [
+                            partial_name[0]
+                            for i in self.package
+                            for partial_name in i.split("_")
+                            if i != self.api_naming.version
+                        ]
                     ),
                     self.module,
                 )

--- a/gapic/schema/metadata.py
+++ b/gapic/schema/metadata.py
@@ -115,7 +115,9 @@ class Address:
             return '_'.join(
                 (
                     ''.join(
-                        i[0]
+                        ''.join(
+                            [partial_name[0] for partial_name in i.split("_")]
+                        )
                         for i in self.package
                         if i != self.api_naming.version
                     ),

--- a/tests/unit/schema/test_metadata.py
+++ b/tests/unit/schema/test_metadata.py
@@ -184,6 +184,14 @@ def test_address_name_builtin_keyword():
     )
     assert addr_kword.module_alias == "gp_class"
 
+    addr_kword = metadata.Address(
+        name="Class",
+        module="class",
+        package=("google", "appengine_admin"),
+        api_naming=naming.NewNaming(proto_package="foo.bar.baz.v1"),
+    )
+    assert addr_kword.module_alias == "gaa_class"
+
 
 def test_doc_nothing():
     meta = metadata.Metadata()

